### PR TITLE
Update cs-rin-ru-enhanced-mod.user.js

### PR DESCRIPTION
**Pull request by Altansar69**
• Colorize frind now works with mentionify and quotes
• Creation of a let storing the colors used by the script

### CODE REVIEW

File: cs-rin-ru-enhanced-mod.user.js [hunk 1]
```diff
@@ -487,10 +495,8 @@ function setupTopicPreview() {
         const topic = $(e)[0];
         let tid;
         $(topic).off("mouseover").on("mouseover", () => {
-            showPreview = true;
```
Comment (SubZeroPL) [inline cs-rin-ru-enhanced-mod.user.js:490]: Actually I think this variable is needed because without it in some specific conditions (it's hard to reproduce) when you hover mouse over topic and then move it away, the preview pops up anyway.

Comment (Reddiepoint) [reply cs-rin-ru-enhanced-mod.user.js:490]: I don't know if I'm experiencing the same situation as you, but I seem to get it with or without this variable anyway. I think it has something to do with the logic of loading the preview. Currently, the code waits for the user to hover over the topic for a certain amount of time, and then it will get the page, which also takes some time. 

I think the problem occurs when the user hovers over the topic over the specified amount of time and then moving the mouse away, which would cause the code to load the page because the timer has expired. 

Basically, if I set the timer to 1 second, and loading the page takes 2 seconds, if I hover over the link for 1 second, the timer will expire and the `setTimeout` code will run, and once the `GM_xmlhttpRequest` is sent, it can't be cancelled. I did some testing, and it seems that hovering over tags seems to lead to unintended behaviour (like loading the wrong page, sending multiple requests and loading the tags of other topics the user hovers over).

Comment (SubZeroPL) [reply cs-rin-ru-enhanced-mod.user.js:490]: > I think the problem occurs when the user hovers over the topic over the specified amount of time and then moving the mouse away, which would cause the code to load the page because the timer has expired.

That's what this variable was for, so after the page preview is loaded we check again if user still keeps mouse cursor over topic.
Don't know about tags, they should be a part of topic title so they should work in the same way.

Comment (SubZeroPL) [reply cs-rin-ru-enhanced-mod.user.js:490]: Anyway current version seems to work fine.

**Review by SubZeroPL — APPROVED** [review]

**Comment by SubZeroPL** [comment]
Added small fix to readme file

**Review by SubZeroPL — APPROVED** [review]

File: cs-rin-ru-enhanced-mod.user.js (hunk 1 above)
Comment (Reddiepoint) [reply cs-rin-ru-enhanced-mod.user.js:490]: > That's what this variable was for, so after the page preview is loaded we check again if user still keeps mouse cursor over topic. Don't know about tags, they should be a part of topic title so they should work in the same way.

Yeah, I just modified the code a little so it should work as expected.
